src/blocked.c

```diff
@@ -111,7 +111,8 @@ void blockClient(client *c, int btype) {
 void updateStatsOnUnblock(client *c, long blocked_us, long reply_us){
     const ustime_t total_cmd_duration = c->duration + blocked_us + reply_us;
     c->lastcmd->microseconds += total_cmd_duration;
-
+    if (server.latency_tracking_enabled)
+        updateCommandLatencyHistogram(&(c->lastcmd->latency_histogram), total_cmd_duration*1000);
```

Comment:
If in practice we only support μsecs then why did you write the whole things with nsecs as the base unit?

Reply:
@yoav-steinberg it is to get more precision on the start of the histogram. otherwise, given we have operations that take below 1 micro they would not properly show up.